Client/Frontend/Widgets/SimpleTwoLineCell.swift

```diff
@@ -0,0 +1,104 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+// TODO: Remove SnapKit
+class SimpleTwoLineCell: UITableViewCell, NotificationThemeable {
```

Comment:
If not used in the project, can we just remove it entirely?

Reply:
Yes I just feedback of the rest of the team